guides/source/action_view_overview.md

```diff
@@ -517,43 +556,50 @@ CAUTION: Only keyword arguments are supported. Defining positional or block argu
 Layouts
 -------
 
-Layouts can be used to render a common view template around the results of Rails controller actions. Typically, a Rails application will have a couple of layouts that pages will be rendered within. For example, a site might have one layout for a logged in user and another for the marketing or sales side of the site. The logged in user layout might include top-level navigation that should be present across many controller actions. The sales layout for a SaaS app might include top-level navigation for things like "Pricing" and "Contact Us" pages. You would expect each layout to have a different look and feel. You can read about layouts in more detail in the [Layouts and Rendering in Rails](layouts_and_rendering.html) guide.
+Layouts can be used to render a common view template around the results of Rails controller actions. A Rails application can have multiple layouts that pages can be rendered within. 
+
+For example, an application might have one layout for a logged in user and another for the marketing part of the site. The logged in user layout might include top-level navigation that should be present across many controller actions. The sales layout for a SaaS app might include top-level navigation for things like "Pricing" and "Contact Us" pages. Different layouts typically have a different header and footer content. 
+
+To find the layout for the current controller action, Rails first looks for a file in `app/views/layouts` with the same base name as the controller. For example, rendering actions from the PhotosController class will `use app/views/layouts/photos.html.erb` (or `app/views/layouts/photos.builder`). 
+
+If such a controller-specific layout does not exist, Rails will use `app/views/layouts/application.html.erb` (or `app/views/layouts/application.builder`). Rails also provides more ways to assign specific layouts to individual controllers and actions. See [Layouts and Rendering guide](layouts_and_rendering.html#finding-layouts) for more.
+
+You can learn more about layouts in general in the [Layouts and Rendering in Rails](layouts_and_rendering.html) guide.
 
 ### Partial Layouts
 
 Partials can have their own layouts applied to them. These layouts are different from those applied to a controller action, but they work in a similar fashion.
 
-Let's say we're displaying an article on a page which should be wrapped in a `div` for display purposes. Firstly, we'll create a new `Article`:
+Let's say we're displaying an article on a page which should be wrapped in a `div` for display purposes. First, we'll create a new `Article`:
 
 ```ruby
 Article.create(body: 'Partial Layouts are cool!')
 ```
```

Comment:
We might be able to skip this "create an article" part, since we don't do anything similar in previous sections, we just assume there's this `@variable` in the template/partial.

Reply:
Just a note from a beginner's POV - creating the article did help me understand this better.

Reply:
My thinking is that this is one of the last examples of the guide, but the first one that actually shows the object creation -- all others simply infer there's an object / instance variable and use them in the many "render" examples (like `@product`, etc.).

Definitely don't mind keeping it, just felt slightly inconsistent with the rest of the examples. If we were to intro the guide with something like "let's create an object and use it as an example throughout the guide", that could make the flow more natural and consistent. Alternatively, we could replace the "article" usage in this example with "product" since we've been using that, and I think the same partial layout example would still make sense.